conn.go

```diff
@@ -678,6 +678,11 @@ func (w *writeCoalescer) writeFlusher(interval time.Duration, quit chan struct{}
 		<-timer.C
 	}
 
+	go func() {
+		<-quit
+		w.fcond.Broadcast()
```

Comment:
Create a method called Close on writeCoalescer like

 ```go
func (w *writeCoalescer) Close() {
    w.fcond.L.Lock()
    defer w.fcond.L.Unlock()
    if !w.running {
        w.fcond.Broadcast()
     }
}
```

Reply:
Pretty sure locking here won't work. Report #1200 shows that there's another goroutine waiting in `writeFlusher` in conn.go#685, which means the w.fcond.L lock has been acquired and will not allow another one. So, if the connection is closed, pretty sure just broadcasting would be enough.

Reply:
@Zariel having two closing mechanism (quit channel, and closer implementation) sounds pretty confusing two me. Should we really keep both?

Reply:
@AlexisMontagne pretty sure it's covered now by https://github.com/gocql/gocql/blob/master/conn.go#L708